### PR TITLE
fix: wui-avatar gradient algorithm

### PR DESCRIPTION
**Pull request by tomiir**
# Breaking Changes

N/A

# Changes

- feat:
- fix: avatar looking weird when border-radius was customized via `w3m-border-radius-master` as it's intended to be round.
- chore:

# Associated Issues

closes [#1707](https://github.com/WalletConnect/web3modal/issues/1707)

### CODE REVIEW

**Review by 0xAsimetriq — COMMENTED** [review]
We ideally want to keep this connected to theme's border radius as dapps might have brutalist / squarish design needs. Plus this will make ens avatars rounded all the time as well. What if you were to adjust algorithm that produces default avatar image instead or "scale it" up proportionally to border radius so it doesn't have those weird edges in square frame?

**Comment by tomiir** [comment]
Came up with something that might work! 
https://github.com/WalletConnect/web3modal/assets/12738697/6e353279-56c2-41bd-bc54-298512304712

**Review by arein — DISMISSED** [review]
LGTM but I don't know CSS enough and it seems the commented out stuff can be removed?

File: packages/ui/src/composites/wui-avatar/styles.ts [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default css`
   :host([data-variant='generated']) {
     box-shadow: 0 0 0 8px var(--wui-gray-glass-005);
     background: radial-gradient(
-      75.29% 75.29% at 64.96% 24.36%,
+      /* 75.29% 75.29% at 64.96% 24.36%, */ var(--local-radial-circle),
```
Comment (0xAsimetriq) [inline packages/ui/src/composites/wui-avatar/styles.ts:55]: If this comment is no longer needed, let's remove

**Review by 0xAsimetriq — APPROVED** [review]

**Review by arein — APPROVED** [review]
LGTM!

**Pull request merged**